### DISCUSSION TITLE
Sending HL7 Message from Erba XL-180 machine to a mirth channel

### DISCUSSION THREAD

**Discussion by CH-LOVERS-Technologies** [nextgenhealthcare/connect#5353] [Q&A]
I'm trying to get HL7 message from a biochemistry laboratory machine, Erba XL-180 to a mirth channel but i endup getting **???????** as a content of the message mirth mirth channel. I have set HL7v2.x as a data type for inbound, TCP Listerner as a connector, TCP as Transmission mode, the encoding type is UTF-8. Help please...!

**Comment by siddharthsharma9258** [comment]
Well even before a machine can send the data, it sends handshake characters, which would be in the device interfacing manual. These question mark are those handshake characters.

**Reply by CH-LOVERS-Technologies** [reply]
How could i disable it, coz i get those question marks as a message content at every time i try to send the report to LIS

**Reply by siddharthsharma9258** [reply]
You cannot. 
Device interface work differently. While the device is attached constantly, it does not constantly send data.  It only sends when there is a reading performed. Before sending, it checks if the other party is ready or not, and performs handshake. 
During the handshake, there are certain characters that **you (receiver)** have to send back, to confirm that you are ready and alive to accept the data. 
Mirth channel may not be able to do it, because it has to be dealt character by character. If someone has done it somehow, then I don't know.
